### PR TITLE
fix: Use markdown for README

### PR DESCRIPTION
**Pull request by bevsxyz**
I have made some visual changes to make the readme more compatible with Github rendering.

### CODE REVIEW

**Comment by okdana** [comment]
I'm not personally opposed to re-arranging these files, but it's a big change to a very long-standing convention, so i think it needs to be discussed on the mailing list. Could you post your proposal there (`zsh-workers at zsh.org`)?

**Comment by larryv** [comment]
> I'm not personally opposed to re-arranging these files

I am. I don't see why we should markup-ify the readme for the benefit of viewing it on a platform that we don't even officially use.

**Comment by bevsxyz** [comment]
I have added the following version badge that gets its version from the latest tag data provided by Github. This is in the JSON format, which is queried with https://shields.io/ to generate the badge. The badge version updation will be independent of the README.md source file.

[Version](https://github.com/zsh-users/zsh/tags)

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# THE Z SHELL (ZSH)
+
+# <img src="https://www.zsh.org/color_vertical_icon.png" alt="ZSH" width="50"/> THE Z SHELL (ZSH)
```
Comment (larryv) [inline README.md:2]: Our actual project host, SourceForge, [strips raw HTML when rendering Markdown](https://sourceforge.net/p/forge/documentation/Files-Readme/), so this won't work.

I remind you, again, that this project is not hosted on GitHub.

**Comment by bevsxyz** [comment]
I have created a fork of zsh in Sourceforge just to see for myself how the readme looks. It looks fine.

https://sourceforge.net/p/zsh-fork/code/ci/master/tree/

File: README.md (hunk 1 above)
Comment (bevsxyz) [reply README.md]: I have removed raw HTML and replaced it with a markdown image tag without styling.

FYI Sourceforge does not strip HTML. I may now have to revert the changes 😮‍💨 

https://sourceforge.net/p/zsh-fork/code/ci/a205af4809fbe2335438e6b83c1b55bdabba5504/tree/

Comment (larryv) [reply README.md]: > FYI Sourceforge does not strip HTML.

I guess their documentation is wrong, or I misunderstood it.

Comment (bevsxyz) [reply README.md]: That's okay. I will revert back to raw HTML only if we choose to go ahead with the PR. I hope that's alright.

**Pull request closed without merging**